src/components/DataTable/index.js

```diff
@@ -52,10 +63,25 @@ export function DataTable ({
 		}, 500);
 	}, [filterValue]);
 
+	// rounding page number divisible by 5
+	function roundedNumRows(x) {
+		if (Number.isInteger(x)) {
+			const rowCount = Math.round(x / 5) * 5;
+			if (rowCount <= 0) {
+				return 5;
+			} else {
+				return rowCount;
+			}
+		} else {
+			return limit;
```

Comment:
@Aringoaway Are you sure about this? Would not it be overwritten by `limit` props passed to `DataTable` component as you have it in the example in the documentation? And since in example you pass `undefined`, it can happen, that this will return `undefined` as well... and I dont think this is desired

Reply:
@Aringoaway

https://github.com/TeskaLabs/asab-webui/pull/356#issuecomment-1245032113

This is happening exactly because of that you are returning `undefined` as I described in my comment in this thread

Reply:
You should return `default` value here and the default is 10 if I am not mistaken...